### ISSUE TITLE
Recognition accuracy should be specified in image accessible description.

### ISSUE THREAD

**Issue by yzen** [mozilla/Bubo-Eye#14]
Accuracy percentage is still relevant to the user when they discover possible text. We should not ignore it and instead, we should insert that information into a hidden node that we refer to by its id from aria-describedby set on the image element.
